extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/photos/GooglePhotosInterface.java

```diff
@@ -238,16 +254,19 @@ <T> T makePostRequest(
 
   private HttpResponse handleHttpResponseException(
       SupplierWithIO<HttpRequest> httpRequest, HttpResponseException e)
-      throws IOException, InvalidTokenException, PermissionDeniedException {
+      throws IOException, InvalidTokenException, PermissionDeniedException, UploadErrorException {
```

Comment:
Oh i see its because we do the Exception translation here and this is used by every other function. Is it possible to handle the UploadError separately in the `uploadPhotoContent` function?